src/EVM/SymExec.hs

```diff
@@ -14,15 +14,16 @@ import Control.Monad.State.Strict (runStateT)
 import Data.Bifunctor (second, first)
 import Data.ByteString (ByteString)
 import Data.ByteString qualified as BS
+import Data.Maybe (mapMaybe)
```

Comment:
There is already `import Data.Maybe` a few lines below. I think this belongs there, no?

Reply:
Good point, sorry. Fixed.